src/Germany/README.md

```diff
@@ -101,6 +105,29 @@ var clientId = client.SuccessResult.Id;
 
 **Creation of a new Tss id**
 ```csharp
-var tss = await client.CreateTssAsync(accessToken, TssState.Initialized, description: "Creating a test TSS.");
+var tss = await client.CreateTssAsync(accessToken);
 var tssId = tss.SuccessResult.Id;
 ```
```

Comment:
Pls also show getting the PUK in code and ideally add a comment in the test itself, that it should be stored somewhere.

Reply:
it is exactly below ur comment :D

Reply:
`in code`. Also `comment in test itself`